app/page.tsx

```diff
@@ -29,26 +30,52 @@ import {
 } from './filters';
 import { isAddress } from 'viem';
 import Image from 'next/image';
+import { getMarketCategories, getOpenMarkets } from '@/queries/dune';
+import { Categories } from '@/constants';
 
 const ITEMS_PER_PAGE = 12;
 const SEARCH_DEBOUNCE_DELAY = 600;
 const DEFAULT_ORDER_OPTION = orderFilters[0];
 const DEFAULT_STATE_OPTION = stateFilters[0];
 const DEFAULT_CREATOR_OPTION = creatorFilters[0];
 
-enum Categories {
-  TECHNOLOGY = 'technology',
-  CRYPTO = 'crypto',
-  BUSINESS = 'business',
-  POLITICS = 'politics',
-  ECONOMY = 'economy',
-  INTERNATIONAL = 'international',
-  SPORTS = 'sports',
-  ENTERTAINMENT = 'entertainment',
-}
-
 type CategoryOptions = Categories | '';
 
+const groupOpenMarketsByCategory = (data: any) => {
```

Comment:
I think this makes a better name: `rankCategoriesByMarketCount`, it emphasizes that the categories are being ranked (sorted) based on the number of markets.

```suggestion
const rankCategoriesByMarketCount = (data: any) => {
```